Tanasee/Tanasee142.xml

```diff
@@ -26,16 +26,273 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
-                                <support><p/></support>
+                              <support>
+                                 <material key="parchment"/>
+                              </support>
+                              <extent>
+                                <measure unit="leaf">159</measure>
+                                <dimensions type="outer" unit="mm">
+                                  <height>340</height>
+                                  <width>250</width>
+                                </dimensions>
+                              </extent>
+                            <foliation><locus target="#1r">F. 1r</locus> and <locus target="#159">f. 159</locus> left blank.</foliation>
                             </supportDesc>
+
+
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="26">
+                                </layout>
+                            </layoutDesc>
                         </objectDesc>
+
+                        <handDesc>
+                          <handNote script="Ethiopic" xml:id="h1">
+                                <desc>The writing is datable to the <date notBefore="1400" notAfter="1499" resp="bm:Six1999VOHD3">15th Century</date>.</desc>
+                          </handNote>
+                              </handDesc>
+
+<decoDesc>
+<summary>There are sketchy and unfinished drawings throughout the manuscript, some of them are listed below.</summary>
+
+<decoNote type="frame" xml:id="d1">
+<locus target="4r"/>
+<desc>A decorative border divided into four fields.</desc>
+</decoNote>
+
+<decoNote type="drawing" xml:id="d2">
+<locus target="#4v"/>
+<desc>The image of <term key="hodegetria">Virgin Mary</term> with child flanked by angels.</desc>
+</decoNote>
+
+<decoNote type="drawing" xml:id="d3">
+<locus target="#50v"/>
+<desc>St Fāsiladas</desc>
+</decoNote>
+
+<decoNote type="drawing" xml:id="d4">
+<locus target="#78r"/>
+<desc>An angel (?).</desc>
+</decoNote>
+
+<decoNote type="drawing" xml:id="d5">
+<locus target="#129v"/>
+<desc>A man holding a horse. The description reads: <q xml:lang="gez">ዝፈረስ፡ ለተላፊኖስ፡</q>.</desc>
+</decoNote>
+
+</decoDesc>
+
+<additions>
+<list>
+
+<item xml:id="a1">
+<locus from="2v" to="3r"/>
+<desc type="GuestText"><ref type="work" corresp="LIT2444Temher"></ref></desc>
+<note>A wirting exercise. The text is incomplete.</note>
+</item>
+
+<item xml:id="a2">
+<locus from="2r" to="2v"/>
+<locus from="158r" to="158v"/>
+<desc type="GuestText"><ref type="work" corresp="LIT1960Mashaf"></ref></desc>
+<note>A wirting exercise. The text is incomplete.</note>
+</item>
+
+<item xml:id="a3">
+<locus target="#158r"/>
+<desc type="OwnershipNote">The note states that the manuscript belonged to the monastery of <placeName ref="LOC2422DagaEs">Dāgā ʾƎsṭifānos</placeName></desc>.
```

Comment:
we can use the institution ID INS0329DE since it definitely refers to this place as a repository of manuscripts

Reply:
ok, will remember to use INS rather then geographical location